app/static/css/cursors.css

```diff
@@ -0,0 +1,27 @@
+.cursor_disabled {

```

Comment:
Can we change these to dash separators instead of underscore separators (i.e., `.cursor-disabled`)?

Reply:
Yes that's an easy change.